owrx/source/hydrasdr.py

```diff
@@ -0,0 +1,60 @@
+from owrx.source.soapy import SoapyConnectorSource, SoapyConnectorDeviceDescription
+from owrx.form.input import Input, CheckboxInput
+from owrx.form.input.device import BiasTeeInput
+from owrx.form.input.validator import Range
+from typing import List
+
+# case sensitive, be careful changing it
+class HydrasdrSource(SoapyConnectorSource):
+    def getSoapySettingsMappings(self):
+        mappings = super().getSoapySettingsMappings()
+        mappings.update(
+            {
+                "bias_tee": "biastee",
+                "bitpack": "bitpack",
+            }
+        )
+        return mappings
+
+    def getDriver(self):
+        return "hydrasdr"
+
+
+class HydraSDRDeviceDescription(SoapyConnectorDeviceDescription):
```

Comment:
This class name is not in line with the rest of the code. The class names must convert from the snake_case name you use in the configuration ("hydrasdr") into CamelCase. This class must therefor be named `HydrasdrDeviceDescription`.

Please update.